index.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Bierproductie: Info Pages</title>
+    <link rel="stylesheet" href="styles/main.css"/>
+    <script src="scripts/script.js" defer></script>
+    <script src="scripts/maintain.js" defer></script>
+    <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
+  </head>
+  <body>
+
+    <!-- Side navigation -->
+    <div class="sidenav">
+      <span class="iconify marked-page" onclick="location.href='/';" data-icon="cil:touch-app" data-inline="false">
+      </span>
+      <span class="iconify" onclick="location.href='/settings.html';" data-icon="carbon:settings" data-inline="false">
+      </span>
+    </div>
+
+    <!-- topbar -->
+    <header>       
+      <h2>Bierproductie</h2>
+      <div class="batch" style="background-color: transparent;">
+        <p id="batch">BATCH</p>
+        <input class="batchNr" type="text" size="4" value="3425">
+        <div class="custom-select" style="width:60px;">
+          <p id="batchNr"></p>
+          <div class="select-items select-hide">
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <!-- MAIN PAGE START HERE -->
+    <div class="main">
```

Comment:
I like the use of flex boxes, but we might need to put some constraints on them to avoid situations like in the picture below. (I know that the PLC screen will not be a 1440p 25" monitor, but it still needs to be optimised for multiple screen sizes).

![image](https://user-images.githubusercontent.com/55458479/99437467-c9663180-2912-11eb-98e0-3b564ce02f6f.png)

Reply:
Yes that should just be a max with :)